lib/Tracy/DIPanel/DIPanel.php

```diff
@@ -0,0 +1,150 @@
+QI6Hb85108K+ub07v5l8cd7slGz58pAow/CADSG/8QI6Cb8d/2C6PZ4Orpk/rcB/aZlm37mVERsPnxf6vnPggA49+y8Q8pAroZ/yVfmZdNXdLZO971zu7Ntvn4XBFg/EEAEMb4zxqblkwEdPtt/1knDHb3jnmBR8DCa+caf0AApDD+xRhf8b1V2YIdBlsfAd2O/8KP5OPYW8Lb5gYaAYtuHMn65/cZf0AAtH38j33P/x/htkdAGb/qN33nqaV97qFFgPEHBECC49/2CChj/JfcNpb19Jf7uYcSAQ2N/6me+yAAjH/d43/ppke3bRFQ1ov8LLyump+NNx0Bxh8QAMa/dRFQ5iv8jb5jZmWfe/FaAk1EgPEHBIDxb10ElP3yvnPPGq70c687Aow/IAASGf9jLpn8uMYaAVW8tv+CD86p/HOvKwKMPyAAjH/rIqCqN/ZZfMu8bErPlOgjwPgDAiCV8b94WddfUccSAVW/q1/xRj91fO7jEfCXI8YfEADGv7nxjyUC6nhL35FzZ9b2uZcdAcYfEADGv3URUMf4T5ix17ToIsD4AwKgmfGfU/f4H33Rssp+rz60CKhz/MffCfDTo1nvYE80EWD8AQHQXADc2JbxDy0C6h7/CfMunZVN6ZtSbwRcP2L8AQEQ0fgfXudIrnn3stpeW7/pCGhq/J+PgIvDjgDjDwiA5sZ/Zu5nbRz/piOg6fEPPQKMPyAAmg2AI9o8/k1GwMvOCedd9EKLAOMPCIDmA+CPaxn/dzU3/k1GwNwzhkWA8QcEQJAB8LUUxl8ENBQBc18YAcYfEADhBMCTVR7Aq9+8KJjx3zACRrafIQJqjgDjDwiAsALg/ioP4YGZ/dkFX9kjqAC48OG9s5d9cV42dUmfCKgxAow/IADCCoCPVn0YhxQBxfjv8I35//5iOZ8fFQE1RoDxBwRAWAHw5joO5RAiYMPxf/4V80RA2yLA+AMCYCsDYGnuN22PgJcafxHQuggw/oAAmGQEXFbXId1EBGxu/EVAayLA+AMCoIMAmJr7YRsjYGvGXwREHwHGHxAAXUTAbrmn2xQBkxl/ERBtBBh/QACUEAGHtSUCOhl/ERBdBBh/QACIgHLGXwREEwHGHxAAIqDc8RcBwUeA8QcEgAioZvxFQLARYPwBASACqh1/ERBcBBh/QACIgHrGf8MI6N9WBBh/QACIgCAioI7xLyy8dm7WO9xT+/CJAOMPCAARkNj4hxgBc88cNv6AABAB1UfAebfsnvT4hxQBi24c8Za+gAAQAfWMwNTpvS+KgNTGP4QIMP6AAKDRHwekOv5NRoDxBwQAjUXAjOG+7KQv7pz0+DcRAcYfEAA0HgG9gz3Zgg/PSXr864wA4w8IAJKIgFjGv44IMP6AACCJCIht/KuMAOMPCADCjoCr5yQ9/lVEQEPjf5bnECAA4g2A/ty6Ooevd6D7CIh9/MuMAOMPCAA6Gf+bmxi+biKgLeNfRgQYf0AAEM34dxMBbRv/biLA+AMCwJ0Q3fh3EgFtHf9OIsD4AwiAaMd/MhHQwPgXfzHy5NxPQouAhsb/TM8fQAAY/2oi4ENzQhr/w9bfZwtDigDjDyAAWjP+E3peIgKaHP8N7rsgIsD4AwiA1o3/CyLgqjnBjH8oEWD8AQRAa8d/wwgYOWc4mPFvOgIaGv8zPH8AAWD8264Y/8O38j6tPQJ6Z/UafwABYPybGv8mI8D4AwgA49/g+Lc4Aow/IACMv/FPLAKMPyAAjH8y4/+6ku7z2COgGP/TPX8AAWD8J2NdnW8lHNr4tyACjD8gAIz/pN28/s89LKIIKH38I44A4w8IAOPf2fhv8DHEEAGVjX+EEWD8AQFg/Lsb/0gi4JncETU9JqFHQDH+pzk4AAFg/Lse/8AjoLbxjyACjD8gAIx/ueMfaATUPv4BR4DxBwSA8a9m/AOLgMbGP8AIMP6AADD+1Y5/IBFQjP+RgTxmTUeA8QcEgPGvZ/wbjoBgxj+ACCjG/60OCUAAGP/axr+hCAhu/BuMAOMPCADj38z41xwBwY5/AxFg/AEBYPybHf+aIiD48a8xAow/IACMfxjjX3EEFON/VGSPbVURUIz/qQ4FQAAY/2DGv6IIiG78K4wA4w8IAOMf5viXHAHRjn8FEWD8AQFg/MMe/5IiIPrxLzECjD8gAIx/HOO/wef8yg7G7x9zq1v2XZ8iAr7XweP3q9xbHAKAADD+0Yz/Bp/79Nyf5H67hY/3d7mrcgMt/kufl6z/7sbWPH7355Y7AAABYPyjG/+N7offy12W+0ruZ+s/zp/nvpZ7b26PJC7mKVN2za3L/XoTj90TufNzvZ78gAAw/lGP/ybum6GkL+p/vzZ2Kb7Fnzs3d3Buric8QOQBYPwBILEAMP4AkFgAGH8ASCwAjD8AJBYAxh8AEgsA4w8AiQWA8QeAxALA+ANAYgFg/AEgsQAw/gCQWAAYfwBILACMPwAkFgDGHwASCwDjDwCJBYDxB4DEAsD4A0BiAWD8ASCxADD+AJBYABh/AEgsAIw/ACQWAMYfABILAOMPAIkFgPEHgMQCwPgDQGIBYPwBILEAMP4AkFgAGH8ASCwAjD8AJBYAxh8AEgsA4w8AiQWA8QeAxALA+ANAYgFg/AEgsQAw/gCQWAAYfwBILACMPwAkFgDGHwASCwDjDwCJBYDxB4DEAsD4A0BiAWD8ASCxADD+AJBYABh/AEgsAPJbn/EHgPQC4CLjDwAJBUB+W5F72vgDQCIBsP5b/981/gCQVgAcZfwBIL0AuNz4A0B6AfB14w8A6QXAU8YfANILgH8x/gCQXgDcZvwBIL0A+GPjDwDpBcBhxh8A0guAntw64w8ACQXA+gpYlvu/xh8AEgqA9RFwrvEHgPQCoGd9BEzmOwFPr38XwT53JgBEGAAbhMCyrfw7AcUbCK1wJwJACwJgg+8GHLb+VwRvW/9iQU+tf9ngy9e/gZCv+gEgUv8PWRDLGr0PBckAAAAASUVORK5CYII="
```

Comment:
Could you please use this downscaled 30x30 image as a source:

`data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAAAXNSR0IArs4c6QAAAAlwSFlzAAAN1wAADdcBQiibeAAABCRpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IlhNUCBDb3JlIDUuNC4wIj4KICAgPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICAgICAgPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIKICAgICAgICAgICAgeG1sbnM6dGlmZj0iaHR0cDovL25zLmFkb2JlLmNvbS90aWZmLzEuMC8iCiAgICAgICAgICAgIHhtbG5zOmV4aWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20vZXhpZi8xLjAvIgogICAgICAgICAgICB4bWxuczpkYz0iaHR0cDovL3B1cmwub3JnL2RjL2VsZW1lbnRzLzEuMS8iCiAgICAgICAgICAgIHhtbG5zOnhtcD0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLyI+CiAgICAgICAgIDx0aWZmOlJlc29sdXRpb25Vbml0PjI8L3RpZmY6UmVzb2x1dGlvblVuaXQ+CiAgICAgICAgIDx0aWZmOkNvbXByZXNzaW9uPjA8L3RpZmY6Q29tcHJlc3Npb24+CiAgICAgICAgIDx0aWZmOlhSZXNvbHV0aW9uPjkwPC90aWZmOlhSZXNvbHV0aW9uPgogICAgICAgICA8dGlmZjpPcmllbnRhdGlvbj4xPC90aWZmOk9yaWVudGF0aW9uPgogICAgICAgICA8dGlmZjpZUmVzb2x1dGlvbj45MDwvdGlmZjpZUmVzb2x1dGlvbj4KICAgICAgICAgPGV4aWY6UGl4ZWxYRGltZW5zaW9uPjMwPC9leGlmOlBpeGVsWERpbWVuc2lvbj4KICAgICAgICAgPGV4aWY6Q29sb3JTcGFjZT4xPC9leGlmOkNvbG9yU3BhY2U+CiAgICAgICAgIDxleGlmOlBpeGVsWURpbWVuc2lvbj4zMDwvZXhpZjpQaXhlbFlEaW1lbnNpb24+CiAgICAgICAgIDxkYzpzdWJqZWN0PgogICAgICAgICAgICA8cmRmOlNlcS8+CiAgICAgICAgIDwvZGM6c3ViamVjdD4KICAgICAgICAgPHhtcDpNb2RpZnlEYXRlPjIwMTk6MDk6MDMgMTM6MDk6MDU8L3htcDpNb2RpZnlEYXRlPgogICAgICAgICA8eG1wOkNyZWF0b3JUb29sPlBpeGVsbWF0b3IgMy44LjU8L3htcDpDcmVhdG9yVG9vbD4KICAgICAgPC9yZGY6RGVzY3JpcHRpb24+CiAgIDwvcmRmOlJERj4KPC94OnhtcG1ldGE+Ci2FV4MAAAUQSURBVEgNtZZ/TJR1HMc/z/3i7uCOH7ETOI4foY4EEUW0McnYLJKkjOamf5i/UvMHqUmFRlt/VZuZpsgGtKitcIpYqzlDy2y2tdzcTE095afoKQoI7OC8e+55Pr2/Ku5iEwiP9/a65/ne8/1+3s/3x+f7fbQ0ftIg9GLwImgG/WDcJcEh3xoiXQCs0VAZyhHj7gqDUHBsSaaBv1lk5ulxWkb3Pww0147TW8iIa+rxqGnzUw3hBal66dwtJa/DzR78fxZ4n9Q4DEEyQDfwg0Cd6fOS5/hV/5y8FL2pcKKWLtxU8u7e4wGFyfmkxrFwqnnodh5XJcDZjPtsr5+STzT59c/ZJHNmlES/u9Q8r0IdAfXGdCuMT5tMph5cVwLDwyiiQ1vAP2ADWBNjkpoXpWjZYqCLKC8BY1YkWn6WnZ3tLi0tZbvd7kL5DSCGfwEQI/AOCAFCq8HPQKTXmJWAlocXLlzILS0tLNTQ0MA5OTliro+Bk0CYi1wOlFjtY1YcWh7Mz89Xmpqa7puKH1mWubi4mPFM0AD0IGiKQaT9hYWFstPpfGSqKAqXl5fztIwM3rCxmKdnzexEvWVAFwxnYVoLfLXf1T4yVVWVKyoqOMHh4N17y/l6dw/XHKjnmAdzvgL1nyhzxOrdn1kQ7Xt+VRynp6fzqVN/3DevrKzk+Hg7V1ZX8x23j6/c6ubWLjev27yVQx6s9nVoawL/0WjeRiykXZnzo4tmFNr0YgYVcy/VVR0nl8tF1dVVVPLe+5Q5K4da29sp3pFILY1XqXrfHo+rvd2JtlPBGSBW/agVKUlUP/3laP/qqjSOnxLKc5fb+fMrufzCBgdbLKG8bXsZ33b3c0nZRzxlagbvqa7hBa+9rup0uh1wEdOTCESKjVo2jU6zc1bRBHVlRRo70sI4d2kc77iYy/tu5HF521yeszaOUyanct2R43y+1cWLly5jjVbrlySpHi4izx+rxw11rE6v2Z48w7ppxis26URVOyVnWamobCIZLWiCXblT9tGlJDc1trXT6W//pKeibHT+3Fm50Xn5e7htBcNui+LcDJQo2/Umzbakadb189Y66LevrpPtaTMteDeZWGUyW3TUzT46dLKVXH0DFPaMgW7v7iXPUZ84kQ4DsVuNOJ9DexyBvaYsMcOyce4KO7mcA4ReU/6mROrv9lHdtkbysEJ/RXRT06891F/jIU24lvxXFVm+phyCodifb4IRNTTBjVpJirfaDL6BXr+ht8NL895ykNGopagYI4VNMtCBTxvJutxEkQWhpF5UqPPjXnEiiTkVPb01ouPDCkN77GamvztaBjJZpaSCLUkUnWAkZYDptuqltlQP9XTdo/4jXtKESuRt9qv+a8rg8I6qp4MvNtRYlGNNFt0cr9s/+e51L02aHUk9Zpl+PHqNrlzoJct8E8ktCvXV9pPcrPyA+mJ4bwwGHOs1DqnwS1ahjT84mc0zV8ZwyksRnFYfy9G7IzgkXceWV81smKITB8FOMGzKDPcSQ3usYnFFa7VSrn6qXncnQabLX3ZSX6OPrPOMJL6Y3D95yO9S9yLoJ6BruOD/55lIp7eBbEzW+4yzDawxaA5imz9hetaghqTqGfcVeC727qBJHNrrwSWwGZQAsVongVRQB74A8SBoEj1dC1qBOE2MQHys2cCgJuAmfLAQrOubCNQG1oCQYAUdKY5Ih1awCgT1cwXxHiuxc4nU2AW+BoHfxSiOn/4FlYT31T4avjkAAAAASUVORK5CYII=`